objects/obj_creation/Create_0.gml

```diff
@@ -614,14 +614,16 @@ advantage[i]="Scavengers";
 advantage_tooltip[i]="Your Astartes have a knack for finding what has been lost.  Items and wargear are periodically found and added to the Armamentarium.";i+=1;
 advantage[i]="Siege Masters";
 advantage_tooltip[i]="Your chapter is familiar with the ins-and-outs of fortresses.  They are better at defending and attacking fortifications.";i+=1;
-advantage[i]="Slow and Purposeful";
-advantage_tooltip[i]="Careful and steady is your chapters doctrine; all infantry have slightly less attack but boosted defences.";i+=1;
-advantage[i]="Melee Enthusiasts";
-advantage_tooltip[i]="Rip and tear! Each Company has an additional Assault Squad.  Your marines and dreadnoughts also have boosted attack with melee weapons.";i+=1;
+advantage[i]="Devastator Doctrine";
+advantage_tooltip[i]="The steady advance of overwhelming firepower is your chapters combat doctrine each company has an additional Devastator squad, all infantry have boosted defence, and heavy weapons have increased attack.";i+=1;
+advantage[i]="Assault Doctrine";
+advantage_tooltip[i]="Your chapter's prefers quick close quaters assault with the enemy each Company has an additional Assault Squad and your marines are more skilled in melee.";i+=1;
```

Comment:
Also, just a note, but I assume that renames may break saved games.
Maybe traits are saved by their index number in the array, but I'm not sure. Maybe test with an old save that has one of these traits, if possible?

Reply:
I expect that if you have a custom chapter saved with Melee enthusiasts or Slow and purposeful it will wig out